com.unity.sg2/Editor/GraphDeltaRegistry/GraphDelta/GraphHandler.cs

```diff
@@ -115,6 +115,10 @@ public void ReconcretizeAll()
                         {
                             ReconcretizeNode(node.ID.FullPath);
                         }
+                        if (builder.GetRegistryKey().Name == ContextBuilder.kRegistryKey.Name)
+                        {
+                            ReconcretizeNode(node.ID.FullPath);
+                        }
```

Comment:
This may be unnecessary with Liz's changes incoming, but is necessary to ensure port type data propagates correctly from context entries to reference nodes.